### ISSUE TITLE
Request for permission to use your open-source project with quizdown.js and potential compatibility update

### ISSUE THREAD

**Issue by gogoadl** [candidosales/svelte-google-translate#3]
Hello, Mr.@candidosales 

I would like to contribute to other open-source projects using your open-source library. Currently, I am using a service called [ghcertified](https://github.com/FidelusAleksander/ghcertified), which is built with [quizdown.js](https://github.com/gogoadl/quizdown-js) and utilizes Google Translate. However, each time I move to the next quiz, I have to reapply the translation, which led me to your open-source project in search of a solution.

When I tried to use your project with quizdown.js, I encountered a dependency issue in the package.json file, making it difficult to implement. [I was able to resolve this by adding your source code directly to my project.](https://github.com/gogoadl/quizdown-js/commit/30cb1ce88354ebc7dad085f47949684c9540a9e8)

Before I proceed with using your code, I would like to ask for your permission. If this is not allowed, could you possibly make your open-source project compatible with the latest version of quizdown.js?

Your project would be a great help to their service. Thank you!

## use your code on quizdown.js

![녹화_2024_09_13_01_12_49_789](https://github.com/user-attachments/assets/bd82bbb1-04d7-496c-a353-67b5b2ae1ae9)

## before use your code on quizdown.js

![녹화_2024_09_13_01_13_24_76](https://github.com/user-attachments/assets/c0cb0163-9a68-4cef-8a99-8b7a35766d41)

**Comment by candidosales** [comment]
Hi @gogoadl , thanks for reporting.

Which changes should I make to work with [quizdown.js](https://github.com/gogoadl/quizdown-js)?

**Comment by gogoadl** [comment]
Hi @candidosales, Thank you for fast reply!

when i add dependency, i saw this message.

```
PS C:\Users\A93220\Documents\GitHub\quizdown-js> npm i @candidosales/svelte-google-translate
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: quizdown@0.6.0
npm error Found: svelte@3.59.2
npm error node_modules/svelte
npm error   dev svelte@"^3.43.2" from the root project
npm error
npm error Could not resolve dependency:
npm error peer svelte@"^4.0.0" from @candidosales/svelte-google-translate@0.0.15
npm error node_modules/@candidosales/svelte-google-translate
npm error   @candidosales/svelte-google-translate@"*" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error C:\Users\A93220\AppData\Local\npm-cache\_logs\2024-09-13T00_19_39_959Z-eresolve-report.txt
npm error A complete log of this run can be found in: C:\Users\A93220\AppData\Local\npm-cache\_logs\2024-09-13T00_19_39_959Z-debug-0.log
```
and when i add dependency command with `--force` or `legacy-peer-deps`, then chrome devtools console shos like this.

![image](https://github.com/user-attachments/assets/1207335f-f1a0-442f-a09b-d9eb4b883e32)

I think important log message is `Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'GoogleTranslate')`

another message is about the code i wrote, not your library.

So, could you support svelte 3 version? 

I'm not good at front-end develop, so i can't get the another detail information.

**Comment by candidosales** [comment]
So, in this case, could you upgrade your project to use Svelte 4?

**Comment by gogoadl** [comment]
Hi, @candidosales 

I tried to update the library to version svelte 4, but I couldn't migrate it successfully due to lack of knowledge. 

I followed the migration guide through the link below, but it failed due to various collision points.

migration guide : https://svelte.dev/docs/v4-migration-guide

I think it will be difficult for me to solve this problem, so I will request migration from issue of quizdown.js.

Also, this issue is not your library issue, so I will close it.

Thank you for your help.
